packages/api/src/routes/locationRoute.js

```diff
@@ -78,8 +79,9 @@ router.post('/garden', hasFarmAccess({ body: 'farm_id' }), checkScope(['add:gard
 router.post('/farm_site_boundary', hasFarmAccess({ body: 'farm_id' }), checkScope(['add:farm_site_boundary']),
   modelMapping['farm_site_boundary'], createLocation('farm_site_boundary'));
 
-router.post('/:location_id/organic_history', hasFarmAccess({ params: 'location_id' }), checkScope(['add:organic_history']),
-  organicHistoryController.addEntry);
+
+router.post('/organic_history', hasFarmAccess({ body: 'location_id' }), checkScope(['add:organic_history']),
```

Comment:
foreign key should be passed through body for post req

Reply:
Is there a source that LF uses to define REST approaches? I was following this one: http://microformats.org/wiki/rest/urls